airflow/api_fastapi/core_api/datamodels/assets.py

```diff
@@ -102,6 +102,7 @@ class AssetEventResponse(BaseModel):
 
     id: int
     asset_id: int
+    uri: str | None = Field(alias="uri", default=None)
```

Comment:
if we're to add `uri` here, we'll probably need to add `name` (and maybe `group`?)